libkbfs/bcache.go

```diff
@@ -141,6 +142,18 @@ func (b *BlockCacheStandard) CheckForKnownPtr(tlf tlf.ID, block *FileBlock) (
 	return ptr, nil
 }
 
+// SetCleanBytesCapacity implements the BlockCache interface for
+// BlockCacheStandard.
+func (b *BlockCacheStandard) SetCleanBytesCapacity(capacity uint64) {
```

Comment:
The `init.go` code calls this unconditionally.  Shouldn't this function (or maybe the caller) check that `capacity != 0`?